doc/source/tutorial/csgraph.rst

```diff
@@ -47,9 +47,9 @@ later::
     >>> word_list = [word for word in word_list if len(word) == 3]
     >>> word_list = [word for word in word_list if word[0].islower()]
     >>> word_list = [word for word in word_list if word.isalpha()]
-    >>> word_list = map(str.lower, word_list)
+    >>> word_list = list(map(str.lower, word_list))
```

Comment:
For simplicity, I would prefer to just decode these unicode strings to ascii (`bytes`) and leave the rest essentially unchanged.